### ISSUE TITLE
Address pattern library accessibility issues

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3430]
As part of the IT accessibility policy review, it was found that the NRRD pattern library is not compliant.

- As per the review, we need to add more accessibility language to the pattern library. GSA has some guidance on general language. We also need to add accessibility specifics for each section of the library.

As per trusted tester: 

- https://revenuedata.doi.gov/patterns/visual-styles/color - fails color contrast
- https://revenuedata.doi.gov/patterns/visual-styles/color - keyboard trap (its the top nav so it happens on most pages
- https://revenuedata.doi.gov/patterns/visual-styles/typography ; https://revenuedata.doi.gov/patterns/components/?type=Cards - the headings are used for visual cues and not as actual headings
- Skip to main content only exists on some pages
- Some svgs are hidden, others have a title but not an explanation. I think these svgs would be hidden when used on the NRRD site, but since this is an icon library, then should be described here
- The pages have live regions - it highlights a tiny corner on the lower left. If this is not an actual and useful live region it should be removed
- https://revenuedata.doi.gov/patterns/components/?type=Inputs - keyboard trap in the code section within context toggle
- hover state - there's none for unselected tabs/toggles, and the hover for selected toggles matches the unselected state (it should not - it should be unique)
- Headings - the H1 is alwasy the site title - it should be the page title instead (which might affect other headings on the page
- https://revenuedata.doi.gov/patterns/components/?type=Data%20Visualizations - tables have no names/scopes, cells not properly formatted with headers
- https://revenuedata.doi.gov/patterns/components/?type=Data%20Visualizations - charts need better alt text
- https://revenuedata.doi.gov/patterns/components/?type=Toolbars - drop down list have no label, enable access toggle no keyboard access and no visible focus or hover (I don't think we actually use that function so we can remove it from the library instead of fixing it)
